### PR TITLE
Update zwave_js documentation

### PR DESCRIPTION
**Pull request by marcelveldt**
## Proposed change
Extend the documentation for the new Z-Wave JS integration.

## Type of change

- [X] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [X] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by marcelveldt** [comment]
CC @cgarwood and @MartinHjelmare 
do we also need to include info about the auto-setup of the addon feature ?

File: source/_integrations/zwave_js.markdown [hunk 1]
```diff
@@ -19,13 +19,21 @@ ha_codeowners:
...
+1. The [Z-Wave JS Server](https://github.com/zwave-js/zwave-js-server) is the gateway between your Z-Wave USB stick and Home Assistant. You can run this server separately from Home Assistant so your Z-Wave mesh will keep running if you restart or stop Home Assistant. The Home Assistant Z-Wave JS integration connects to this server with a Websockets connection.
+
+2. The Z-Wave JS integration in Home Assistant. Connects to the Z-Wave JS Server running in your network to retrieve the info from your Z-Wave network.
+
 ### Core installation
 
-- The [zwave-js-server](https://github.com/zwave-js/zwave-js-server) installed and running in your network.
+- The Z-Wave JS Server installed and running in your network.
+- We provide an official add-on which you can find in the Supervisor's add-on store.
+- Not running the supervisor ? The [Z-Wave JS 2 MQTT project](https://zwave-js.github.io/zwavejs2mqtt/#/getting-started/quick-start) also includes the Z-Wave JS Server (you can enable it in the settings) and they provide a Docker image.
```
Comment (frenck) [inline source/_integrations/zwave_js.markdown]:
```suggestion
- Not running the Supervisor? The [Z-Wave JS 2 MQTT project](https://zwave-js.github.io/zwavejs2mqtt/#/getting-started/quick-start) also includes the Z-Wave JS Server (you can enable it in the settings) and they provide a Docker image.
```

**Comment by frenck** [comment]
> do we also need to include info about the auto-setup of the addon feature ?

Yes, the PR providing that should be merged for the beta.

File: source/_integrations/zwave_js.markdown [hunk 2]
```diff
@@ -19,13 +19,21 @@ ha_codeowners:
...
 ---
 
-This integration allows you to control a Z-Wave network via Z-Wave JS.
+This integration allows you to control a Z-Wave network via [Z-Wave JS](https://zwave-js.github.io/node-zwave-js/#/).
 
 ## Requirements
 
+Controlling your Z-Wave network with the Z-Wave JS integration is split up into two parts:
+
+1. The [Z-Wave JS Server](https://github.com/zwave-js/zwave-js-server) is the gateway between your Z-Wave USB stick and Home Assistant. You can run this server separately from Home Assistant so your Z-Wave mesh will keep running if you restart or stop Home Assistant. The Home Assistant Z-Wave JS integration connects to this server with a Websockets connection.
```
Comment (cgarwood) [inline source/_integrations/zwave_js.markdown]:
```suggestion
1. The [Z-Wave JS Server](https://github.com/zwave-js/zwave-js-server) is the gateway between your Z-Wave USB stick and Home Assistant. You can run this server separately from Home Assistant so your Z-Wave mesh will keep running if you restart or stop Home Assistant. The Home Assistant Z-Wave JS integration connects to this server with a websocket connection.
```

File: source/_integrations/zwave_js.markdown [hunk 3]
```diff
@@ -19,13 +19,21 @@ ha_codeowners:
...
+
+2. The Z-Wave JS integration in Home Assistant. Connects to the Z-Wave JS Server running in your network to retrieve the info from your Z-Wave network.
```
Comment (cgarwood) [inline source/_integrations/zwave_js.markdown]:
```suggestion
2. The Z-Wave JS integration in Home Assistant. This integration connects to the Z-Wave JS Server to retrieve the info from your Z-Wave network.
```

File: source/_integrations/zwave_js.markdown [hunk 4]
```diff
@@ -46,9 +54,14 @@ available.
 
 As this integration is still in the early stages there are some important limitations to be aware of.
 
-- Polling is not currently supported.
 - Garage door controllers (Barrier Operator CC) are not currently supported.
+- Configuration of Z-Wave nodes and/or configuration with the Home Assistant UI is currently not yet implemented.
 - You will need to use another tool, such as [zwavejs2mqtt](https://github.com/zwave-js/zwavejs2mqtt), to include/exclude devices and manage device configuration.
```
Comment (cgarwood) [inline source/_integrations/zwave_js.markdown]:
```suggestion
- Configuration of Z-Wave nodes and/or configuration with the Home Assistant UI is currently not yet implemented. You will need to use another tool, such as [zwavejs2mqtt](https://github.com/zwave-js/zwavejs2mqtt), to manage device configuration.
```

**Comment by marcelveldt** [comment]
Requested changes are addressed.

**Review by frenck — APPROVED** [review]
Looks good to me! Thanks, @marcelveldt 👍

**Pull request merged**